### PR TITLE
Nem0oo transparent bg

### PR DESCRIPTION
**Pull request by Nem0oo**
Don't know if this works, but it would allow a transparent background for the generated badges

### CODE REVIEW

**Comment by pedrofracassi** [comment]
This won't work, as the frames have white borders around them. We could try to crop the pictures and make them hexagonal, and then remove the border.

**Comment by Nem0oo** [comment]
I already made the border of the frame transparent in my pull request to deal with that ;)

**Comment by pedrofracassi** [comment]
Oh, nice! Now we just need to crop the picture the user submits, so it doesn't show outside the frame

**Comment by Nem0oo** [comment]
Oh, didn't saw that,I'll have a look

File: index.js [hunk 1]
```diff
@@ -42,7 +55,7 @@ bot.on('message', async (msg) => {
       finalCtx.drawImage(pictureCanvas, 125, 125, 710, 710)
       finalCtx.drawImage(frame, 0, 0, 960, 960)
       factions[msg.chat.id] = null
-      bot.sendPhoto(msg.chat.id, finalCanvas.toBuffer('image/png', { quality: 1 }))
+      bot.sendPhoto(msg.chat.id, finalCanvas.toBuffer('image/jpeg', { quality: 1 }))
```
Comment (pedrofracassi) [inline index.js]: why did it go back to jpeg?

**Review by pedrofracassi — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (Nem0oo) [reply index.js]: should have stayed png  :/

**Comment by pedrofracassi** [comment]
Didn't quite work. We'll have to rework this one.

**Pull request merged**